src/tunacode/tools/decorators.py

```
@@ -1,65 +1,79 @@
             return await func(*args, **kwargs)
-        except ToolRetryError as e:
-            raise ModelRetry(str(e)) from e
-        except ModelRetry:
-            raise
-        except ToolExecutionError:
-            raise
-        except FileOperationError:
+        except (ToolRetryError, ToolExecutionError, FileOperationError):
             raise
-        except Exception as e:
+        except Exception as e:  # noqa: BLE001
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)